client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -260,10 +263,10 @@
 			max-width: 95%;
 			margin-top: 0;
 			box-shadow: inset 0 0 1px #e1e1e1;
-			filter: drop-shadow( 0 3px 8px rgba( 0, 0, 0, 0.05 ) ) drop-shadow( 0 3px 3px rgba( 0, 0, 0, 0.03 ) );
+			filter: drop-shadow( 0 3px 8px rgba( 0, 0, 0, 0.05 ) )
+				drop-shadow( 0 3px 3px rgba( 0, 0, 0, 0.03 ) );
```

Comment:
Were these changes made to help with code readability?

Reply:
I didn't make consciously make these changes myself so I think it auto-formatted the file? 🤷‍♂️ 

The updates don't look like anything meaningful so I'll leave it as is for now.